pkg/openstack-cinder/util/const.go

```diff
@@ -0,0 +1,9 @@
+package util
+
+const (
+	// Operand and operator run in the same namespace
+	DefaultNamespace         = "openshift-cluster-csi-drivers"
+	OpenShiftConfigNamespace = "openshift-config"
```

Comment:
So we already have `CSIDriverNamespace` constant in `clients` package in csi-operator, so I would rather reuse that.

Reply:
Ack, done.